eva-accession-core/src/test/java/uk/ac/ebi/eva/accession/core/summary/DbsnpSubmittedVariantSummaryFunctionTest.java

```diff
@@ -160,11 +179,12 @@ public void sameSummaryWhenClusteredVariantDiffers() {
     public void sameSummaryWhenSupportedByEvidenceDiffers() {
         ISubmittedVariant submittedVariant1 =
                 new SubmittedVariant(ASSEMBLY_ACCESSION, TAXONOMY_ACCESSION, PROJECT_ACCESSION, CONTIG, START,
-                                     REF_A, ALT_T, CLUSTERED_VARIANT, SUPPORTED_BY_EVIDENCE, MATCHES_ASSEMBLY);
+                                     REF_A, ALT_T, CLUSTERED_VARIANT, SUPPORTED_BY_EVIDENCE, MATCHES_ASSEMBLY,
+                                     ALLELES_MATCH, VALIDATED);
 
         ISubmittedVariant submittedVariant2 =
                 new SubmittedVariant(ASSEMBLY_ACCESSION, TAXONOMY_ACCESSION, PROJECT_ACCESSION, CONTIG, START,
-                                     REF_A, ALT_T, CLUSTERED_VARIANT, false, MATCHES_ASSEMBLY);
+                                     REF_A, ALT_T, CLUSTERED_VARIANT, false, MATCHES_ASSEMBLY, ALLELES_MATCH, VALIDATED);
```

Comment:
Judging by the rest of declarations, it looks like this one goes over the 120 characters?